app/controllers/events_controller.rb

```diff
@@ -1,3 +1,48 @@
 class EventsController < ApplicationController
-  def index; end
-end
+  def index
+    @events = Event.all
```

Comment:
The `index` action should have a corresponding view to display the events.

Reply:
This is important

Reply:
Done!